aiidalab_widgets_base/viewers.py

```diff
@@ -212,8 +212,12 @@ def __init__(
         if configuration_tabs is None:
             configuration_tabs = ["Selection", "Appearance", "Cell", "Download"]
 
+        self.selection_tab_idx = None
         if len(configuration_tabs) != 0:
-            self.selection_tab_idx = configuration_tabs.index("Selection")
+            try:
+                self.selection_tab_idx = configuration_tabs.index("Selection")
+            except ValueError:
+                pass
```

Comment:
```suggestion
                self.selection_tab_idx = None
```
I'd suggest keeping it more compact.

Reply:
It is set to None by default before the if statement. It must be set there, otherwise if an empty configuration list is passed it would be undefined. But maybe I misunderstood and you want to set it again here?

Reply:
You are not misunderstood, I was suggesting removing the code above. 
I see your point, you are correct 😁